components/server/src/server.rs

```diff
@@ -159,7 +159,7 @@ pub fn run_tikv(config: TiKvConfig) {
 
 const RESERVED_OPEN_FDS: u64 = 1000;
 
-const DEFAULT_METRICS_FLUSH_INTERVAL: Duration = Duration::from_millis(10_000);
+const DEFAULT_METRICS_FLUSH_INTERVAL: Duration = Duration::from_millis(1_000);
```

Comment:
Why change this?

Reply:
`10s` is too long for memory usage increasing. For example, with a heavy write load TiKV can OOM in a less duration than `10s`, so metrics can't record it if the flush interval is `10s`.